tox.ini

```diff
@@ -24,11 +24,12 @@ deps =
     django111: django-filer>=1.3
     cms34: https://github.com/divio/django-cms/archive/release/3.4.x.zip
     cms35: https://github.com/divio/django-cms/archive/release/3.5.x.zip
+    cms36: https://github.com/divio/django-cms/archive/release/3.6.x.zip
     -r{toxinidir}/requirements-test.txt
 
 [testenv:pep8]
 deps = flake8
-commands = flake8
+commands = flake8 {toxinidir}/djangocms_page_meta
```

Comment:
This shouldn't be needed

Reply:
@yakky without this I was getting issues raised in third party packages. This ensures the checks are only ran against this project.

By just running `flake8` tox ends like this;
```
./.venv/lib/python3.7/site-packages/pbr/version.py:207:54: W504 line break after binary operator
./.venv/lib/python3.7/site-packages/pbr/version.py:327:17: W503 line break before binary operator
./.venv/lib/python3.7/site-packages/pbr/util.py:209:13: E117 over-indented
./.venv/lib/python3.7/site-packages/pbr/util.py:211:13: E117 over-indented
./.venv/lib/python3.7/site-packages/pbr/builddoc.py:135:13: E117 over-indented
./.venv/lib/python3.7/site-packages/pbr/packaging.py:543:51: W504 line break after binary operator
./.venv/lib/python3.7/site-packages/pbr/packaging.py:544:40: W504 line break after binary operator
./.venv/lib/python3.7/site-packages/pyflakes/reporter.py:65:70: W504 line break after binary operator
./.venv/lib/python3.7/site-packages/pyflakes/checker.py:1519:40: W504 line break after binary operator
./.venv/lib/python3.7/site-packages/pyflakes/checker.py:1685:53: W504 line break after binary operator
./.venv/lib/python3.7/site-packages/pyflakes/test/harness.py:71:31: W504 line break after binary operator
./.venv/lib/python3.7/site-packages/pyflakes/test/test_doctests.py:31:40: W504 line break after binary operator
./.venv/lib/python3.7/site-packages/pyflakes/test/test_doctests.py:32:46: W504 line break after binary operator
./.venv/lib/python3.7/site-packages/pyflakes/test/test_doctests.py:33:46: W504 line break after binary operator
./.venv/lib/python3.7/site-packages/pyflakes/test/test_doctests.py:34:47: W504 line break after binary operator
./.venv/lib/python3.7/site-packages/pyflakes/test/test_doctests.py:35:44: W504 line break after binary operator
./.venv/lib/python3.7/site-packages/pyflakes/test/test_doctests.py:36:44: W504 line break after binary operator
./.venv/lib/python3.7/site-packages/pyflakes/test/test_api.py:299:50: W504 line break after binary operator
./.venv/lib/python3.7/site-packages/pyflakes/test/test_api.py:300:31: W504 line break after binary operator
ERROR: InvocationError for command '/Users/mwalker/apps/djangocms-page-meta/.tox/pep8/bin/flake8' (exited with code 1)
______________________________________________________________________________________________ summary ______________________________________________________________________________________________
  docs: commands succeeded
ERROR:   pep8: commands failed
```

Reply:
I would avoid to exclude for flake8 every possible virtualenv name, it would make the configuration way less readable
Why not using the tox virtualenv if you need a virtualenv to work with?

Reply:
To be honest I only had a venv because I'm running through Pycharm & it loves to have an interpreter setup.

Reply:
@marksweb I also use PyCharm, latest versions seems to pick up the interpreter from tox dir just fine

Reply:
anyway I think oss projects should be as editor agnostic as possible and avoid adding unneeded things to ease the burden of maintenance